### PR TITLE
Extracts the Dashboard routes

### LINKED ISSUE #2033: Split dashboard routes
The dashboard routes are getting pretty big now, so let's split them up into individual files.

### PR DESCRIPTION
**Pull request by ConnorVG**
Resolves #2033

### CODE REVIEW

**Comment by jbrooksuk** [comment]
Thanks man! I'll check this later.

File: app/Foundation/Providers/RouteServiceProvider.php [hunk 1]
```diff
@@ -68,8 +68,14 @@ protected function registerBindings()
     public function map(Router $router)
     {
         $router->group(['namespace' => $this->namespace], function (Router $router) {
-            foreach (glob(app_path('Http//Routes').'/*.php') as $file) {
-                $this->app->make('CachetHQ\\Cachet\\Http\\Routes\\'.basename($file, '.php'))->map($router);
+            $path = app_path('Http/Routes');
+
+            foreach (glob("{$path}/*{,/*}.php", GLOB_BRACE) as $file) {
```
Comment (GrahamCampbell) [inline app/Foundation/Providers/RouteServiceProvider.php]: Probably a better way to do this. I might just change it after merge.

Comment (ConnorVG) [inline app/Foundation/Providers/RouteServiceProvider.php]: Yeah, I was unsure where to go on this. So I went trusty `GLOB_BRACE` for one/two depth routes. Would probably be nicer going finder but w/e.

Comment (GrahamCampbell) [inline app/Foundation/Providers/RouteServiceProvider.php]: I'll probably make a small package to do this.

Comment (ConnorVG) [inline app/Foundation/Providers/RouteServiceProvider.php]: While you're at it. Make your analysis test trait a package :grin:

Comment (GrahamCampbell) [inline app/Foundation/Providers/RouteServiceProvider.php]: It already is?

Comment (GrahamCampbell) [inline app/Foundation/Providers/RouteServiceProvider.php]: https://github.com/AltThree/TestBench/blob/master/src/AnalysisTrait.php

Comment (ConnorVG) [inline app/Foundation/Providers/RouteServiceProvider.php]: :scream:

**Pull request merged**